<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserirProduto</class>
 <widget class="QWidget" name="InserirProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserir Produto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#InserirProduto{
background-color: rgb(199, 199, 199);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(199, 199, 199, 255), stop:1 rgba(170, 170, 170, 255));

}
QPushButton{
background-color: rgb(43, 132, 191);
color: rgb(255, 255, 255);
font-size: 15px;
border-radius: 13px;

}

QPushButton:hover{
	background-color: rgb(13, 92, 161);
}
QPushButton:pressed{
	
color: rgb(255, 255, 255);
	background-color: rgb(153, 193, 241);
font-size: 15px;
border-radius: 13px;
}
QTabWidget{
background-color:rgb(222, 221, 218)
}
QGroupBox{
background-color:rgb(170, 170, 170);
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 15px;
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_11">
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Cadastrar Produto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Btn_Enviar">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>543</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Principal</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>161</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Quantidade 
Estoque:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomLineEdit" name="Ledit_Quant"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>361</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Descrição:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomLineEdit" name="Ledit_Desc"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Código de Barras:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomLineEdit" name="Ledit_CBarras"/>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_GerarCBarras">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>25</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Gerar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>261</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Preços</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Preço Final:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledit_PrecoFinal"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>221</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Percentual de 
 Lucro(%):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Ledit_PercentualLucro"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>231</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Preço Fornecedor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomLineEdit" name="Ledit_PrecoFornecedor"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>160</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Unidade 
Comercial:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBox_UCom"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Imposto</string>
      </attribute>
      <widget class="QCheckBox" name="Check_Nf">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>NF</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>251</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="toolTip">
           <string>Código Especificador da Substituição Tributária,  você geralmente pode deixar em branco.</string>
          </property>
          <property name="text">
           <string>CEST:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomLineEdit" name="Ledit_CEST"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>251</width>
         <height>47</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="toolTip">
           <string>Porcentagem de ICMS aproximada cobrado de cada produto, APENAS PARA INFORMAÇÃO NA NF.</string>
          </property>
          <property name="text">
           <string>Aliquota Imposto
Aproximada(IBPT)(%) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Ledit_Aliquota"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="Lbl_NcmDesc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Descrição: NCM</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>251</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="toolTip">
             <string>Nomenclatura Comum do Mercosul, pesquise o codigo correto para seu produto.</string>
            </property>
            <property name="text">
             <string>NCM:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CustomLineEdit" name="Ledit_NCM"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>251</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>CSOSN:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Ledit_CSOSN"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>251</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>CST 
PIS/COFINS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Ledit_PIS"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Ledit_CBarras</tabstop>
  <tabstop>Ledit_Desc</tabstop>
  <tabstop>CBox_UCom</tabstop>
  <tabstop>Ledit_Quant</tabstop>
  <tabstop>Ledit_PrecoFornecedor</tabstop>
  <tabstop>Ledit_PercentualLucro</tabstop>
  <tabstop>Ledit_PrecoFinal</tabstop>
  <tabstop>Btn_Enviar</tabstop>
  <tabstop>Btn_GerarCBarras</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>Check_Nf</tabstop>
  <tabstop>Ledit_NCM</tabstop>
  <tabstop>Ledit_Aliquota</tabstop>
  <tabstop>Ledit_CSOSN</tabstop>
  <tabstop>Ledit_PIS</tabstop>
  <tabstop>Ledit_CEST</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
